app/src/main/java/com/yacgroup/yacguide/StatisticsActivity.kt

```diff
@@ -48,21 +48,34 @@ class StatisticsActivity : BaseNavigationActivity() {
     }
 
     private fun _displayContent() {
-        // Placeholders:
+        // The following counts contain all years, even if there was no ascent of the according style.
+        // This means that the list sizes are all equal.
+        val soloCounts = _db.getAscendCountPerYear(AscendStyle.eSOLO.id)
+        val leadCounts = _db.getAscendCountPerYear(AscendStyle.eONSIGHT.id, AscendStyle.eHOCHGESCHLEUDERT.id)
+        val followCounts = _db.getAscendCountPerYear(AscendStyle.eALTERNATINGLEADS.id, AscendStyle.eHINTERHERGEHAMPELT.id)
+        val unknownCounts = _db.getAscendCountPerYear(AscendStyle.eUNKNOWN.id)
+        val otherCounts = followCounts.map { followCount ->
```

Comment:
When I first looked at the statistic, I was wondering why there is no "follow" bar. I would like to separate "unknown" and "follow" bars. Actually, there should no "unknown" bar at all.

Reply:
But how to handle style "Alternating leads" then? (is now part of "others")

Reply:
As discussed offline, it might get to messy to have another style in the diagram. However, "Sonstige" is for me still a bit confusing. "Nachstieg, etc." or so could be an alternative that is a bit more obvious, because "Nachstieg" is one of the most used ascend styles.